webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/resource/visualizer.js

```diff
@@ -607,6 +607,9 @@ var Visualizer = (function($, window, undefined) {
           if (entity[4]) {
         	  span.color = entity[4];
           }
+          if (entity[5]){
+        	  span.hovertext = entity[5];
+          }
```

Comment:
When making changes to the JS files, please always surround the changes with a prominent comment.
The new change does not belong to 

```
// WEBANNO EXTENSION BEGIN - #820 - Allow setting label/color individually
```

but rather to a different issue.

Reply:
True, changed it in 3c8ad3557a4a6f5fa310e60224a17fd81cd2c68b